modules/bookClass.js

```diff
@@ -36,6 +35,7 @@ export class BookClass {
 }
```

Comment:
- [ ] While adding or removing a book your page reloads. You have made a Single Page Application that is not expected to reload. Please fix this issue. :+1: